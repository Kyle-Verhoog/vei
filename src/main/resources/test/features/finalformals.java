// FAIL
// Java 1.3
public class Finalformals {
  public int m(final int x) {
    return x;
  }
}