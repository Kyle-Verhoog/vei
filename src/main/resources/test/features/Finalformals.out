PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
INT ┃ int
IDENTIFIER ┃ m
( ┃ (
FINAL ┃ final
INT ┃ int
IDENTIFIER ┃ x
) ┃ )
{ ┃ {
RETURN ┃ return
IDENTIFIER ┃ x
; ┃ ;
} ┃ }
} ┃ }